github/code-scanning.go

```diff
@@ -18,6 +18,51 @@ import (
 // GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/code-scanning/
 type CodeScanningService service
 
+// Rule represents the complete details of GitHub Code Scanning alert type.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/code-scanning#list-code-scanning-alerts-for-a-repository
```

Comment:
Since all these struct URLs are identical, let's just please delete them to avoid the duplication.  Thanks.